workspace-manager-clienttests/tools/render-config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+VAULT_TOKEN=${1:-$(cat "$HOME"/.vault-token)}
+WM_APP_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/dev/workspace/app-sa
+WM_APP_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/../rendered/workspace-manager-app-service-account.json
```

Comment:
This script failed for me when I first tried to run it, since there isn't a workspace-clienttests/rendered/ directory defined by the git repo. Maybe create one and add a .gitignore file?

Reply:
I'm not sure git will preserve an empty directory. The way I handled that in Buffer Service is to add a `mkdir -p ../rendered` to the script.

Reply:
Added `.gitignore` to revered/ folder.